pocs/utils/images/horizon.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
```

Comment:
I'm thinking about how to produce these from other info, like the azimuth and height above "ground" of the obstruction, and the height above ground of the mount/cameras (probably the lowest height).

Reply:
You mean how to produce a Horizon automatically? That's partly what #364 is about.

Reply:
Not precisely; I meant "get horizon from measurements" as opposed to "get horizon from image". If the latter is going to be easy to achieve, great. But if it will be a while, then I could ask one of the student's I'm working with to help with measuring the space so we can build the former. I could ask him about implementing it.

Reply:
Regarding why this might be useful: the big obstruction next to PAN006 has a constant height, but its altitude at different azimuths will be very different. We could encode it with 5 measurements:
1. Height of wall.
2. Compass angle and distance to start of wall.
3. Compass angle and distance to end of wall.

Reply:
Got it. We should capture that in a wiki page or document so have an Issue in #367.